python/requirements.txt

```diff
@@ -47,38 +47,45 @@ starlette==0.18.0
 aiorwlock
 gradio; platform_system != "Windows"
 
-# Requirements for running tests
-pyarrow >= 6.0.1, < 7.0.0
-# Used for Dataset tests.
-polars
+## Requirements for running tests
+
+# General test requirements
```

Comment:
this comment seems a bit redundant with the one right above.
is this long list of deps all for testing?

Reply:
Over time, there where to categories of tests requirements added (but they where added right inside these "general" requirements. I've split this up into "General test requirements", "For doc tests" and "For dataset tests". They are all not setup requirements though. That's the logic behind the commet. Happy to change - should I?